src/test/java/use_case/game_filter/GameFilterInteractorTest.java

```diff
@@ -0,0 +1,122 @@
+package use_case.game_filter;
+
+import data_access.GameInfoDataAccessObject;
+import org.junit.Test;
+import use_case.GameInfoDataAccessInterface;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class GameFilterInteractorTest {
+
+    @Test
+    public void luckBased_3test() {
+        String gametype = "Luck-Based Games";
+        int playerCount = 3;
+        GameFilterInputData inputData = new GameFilterInputData(gametype, playerCount);
+        GameInfoDataAccessInterface gameInfoDAO = new GameInfoDataAccessObject("game_info.json");
```

Comment:
I think our tests are not supposed to depend on our actual game data. Maybe we can write setter methods in the GameInfoDAO so we can put custom game data in for testing purposes.

Reply:
You are absolutely right. We shouldn't use our actual game data in the tests because any changes to the JSON file would require us to update the corresponding tests as well. However, I believe that using setters to configure the DAO would make our tests harder to read and more difficult to debug when errors occur. Therefore, I have created a dedicated JSON file specifically for testing purposes.